.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: bash
+
+install:
```

Comment:
Here is something missing to make sure that wget is the newest version from precise:

    sudo: required
    install:
      - sudo apt-get update && sudo apt-get install wget

Or just switch to trusty via:

    dist: trusty
    sudo: false